index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
```

Comment:
I believe this is a copy of `card.html` I would eliminate `card.html` and stick with `index.html`